Formula/piknik.rb

```diff
@@ -0,0 +1,54 @@
+class Piknik < Formula
+  desc "Copy/paste anything over the network"
+  homepage "https://github.com/jedisct1/piknik"
+  url "https://github.com/jedisct1/piknik/archive/0.9.tar.gz"
+  sha256 "4b61e78d7c2f4ddc69cffe9e800a6ca03af8e90540a089469def615388087983"
+  head "https://github.com/jedisct1/piknik.git"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/jedisct1/"
+    dir.install Dir["*"]
+    ln_s buildpath/"src", dir
+    cd dir do
+      system "godep", "restore"
+      system "go", "build", "-o", bin/"piknik", "."
+      (prefix/"etc/profile.d").install "zsh.aliases" => "piknik.sh"
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    For Bash or Zsh, put something like this in your $HOME/.bashrc or $HOME/.zshrc:
```

Comment:
Should just be able to use `#{shell_profile}` here rather than `$HOME/.bashrc or $HOME/.zshrc`.